agent/lib/docker/container.rb

```diff
@@ -34,6 +34,12 @@ def restart_policy
       self.host_config['RestartPolicy']
     end
 
+    # @return [String] Stack aware name of the containers service
+    def service_name
+      return self.labels['io.kontena.service.name'] if self.default_stack?
```

Comment:
I think this name is still misleading.. if this is only a lb specific thing maybe name should be something like `lb_service_name` or `service_name_for_lb`? And should this also use `/` as a separator?